.github/workflows/build.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       max-parallel: 4
       matrix:
         # test against latest update of each major Java version, as well as specific updates of LTS versions:
-        java: [8, 8.0.192, 11.0.x, 11.0.3, 12, 13, 15, 16, 17 ]
+        java: [8, 11, 16, 17 ]
```

Comment:
I would keep the list of versions we had

Reply:
Yep, makes sense. I was doing some tests with the new actions/setup-javaV3.